Doc/c-api/type.rst

```diff
@@ -95,6 +95,12 @@ Type Objects
    from a type's base class.  Return ``0`` on success, or return ``-1`` and sets an
    exception on error.
 
+.. c:function:: const char* PyType_GetName(PyTypeObject *type)
+
+   Return the type's name. This function can accept heap or static types.
```

Comment:
> This note is not relevant, since there's no limitation to document.

Oh, I try to address this function can accept any types by this sentence, and removed the note is fine in fact.

> For how long is the returned pointer valid?

How about this note: `You can hold this pointer until the type have been deallocated.`